### ISSUE TITLE
Restart not a valid Permission

### ISSUE THREAD

**Issue by AleksanderRoszig** [CrunchyData/postgres-operator#2203]
**Describe the bug**
When I try update my role and add permission to restart then I got: `Restart not a valid Permission`

Roles: https://github.com/CrunchyData/postgres-operator/blob/72df1a1fd764059fb0c0da7bae0913bfefc194ed/internal/apiserver/perms.go#L32

**To Reproduce**
1. Create role
2. `pgo update pgorole standard --permissions="Restart"Error: Restart not a valid Permission`
3. `pgo update pgorole standard --permissions="Cat,Clone,CreateCluster,CreateBackup,CreateDump,CreateFailover,CreatePgAdmin,CreateSchedule,CreatePgbouncer,CreatePolicy,CreateUpgrade,CreateUser,DeleteBackup,DeleteCluster,DeletePgAdmin,DeletePgbouncer,DeletePolicy,DeleteSchedule,DeleteUser,DfCluster,Label,Reload,Restore,RestoreDump,ScaleCluster,ShowBackup,ShowCluster,ShowConfig,ShowPgAdmin,ShowPgBouncer,ShowPolicy,ShowPVC,ShowSchedule,ShowNamespace,ShowWorkflow,Status,TestCluster,UpdatePgBouncer,UpdateCluster,Version,UpdateUser" pgorole updated` So that mean I can update this role.

and when I add Restart at the end then is again error 
```
pgo update pgorole standard --permissions="Cat,Clone,CreateCluster,CreateBackup,CreateDump,CreateFailover,CreatePgAdmin,CreateSchedule,CreatePgbouncer,CreatePolicy,CreateUpgrade,CreateUser,DeleteBackup,DeleteCluster,DeletePgAdmin,DeletePgbouncer,DeletePolicy,DeleteSchedule,DeleteUser,DfCluster,Label,Reload,Restore,RestoreDump,ScaleCluster,ShowBackup,ShowCluster,ShowConfig,ShowPgAdmin,ShowPgBouncer,ShowPolicy,ShowPVC,ShowSchedule,ShowNamespace,ShowWorkflow,Status,TestCluster,UpdatePgBouncer,UpdateCluster,Version,UpdateUser,Restart"
Error: Restart not a valid Permission
```

**Expected behavior**
The possibility to add a role "Restart" for existing pgorole.

**Please tell us about your environment:**

* Operating System:
* Where is this running ( Local, Cloud Provider): Local
* Storage being used (NFS, Hostpath, Gluster, etc): NFS
* Container Image Tag: registry.connect.redhat.com/crunchydata/postgres-operator:ubi7-4.5.1
* PostgreSQL Version:
* Platform (Docker, Kubernetes, OpenShift):
* Platform Version: Openshift 3.11

**Comment by jkatz** [comment]
If you're effectively granting every permission, I would recommend just using `*`

**Comment by jkatz** [comment]
Merged and backpatched down to the 4.4 series. Thanks for reporting!
